recipes/breakpad/all/conanfile.py

```diff
@@ -0,0 +1,118 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+import os
+import textwrap
+
+required_conan_version = ">=1.33.0"
+
+class BreakpadConan( ConanFile ):
+    name = "breakpad"
+    description = "A set of client and server components which implement a crash-reporting system"
+    topics = ["crash", "report", "breakpad"]
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://chromium.googlesource.com/breakpad/breakpad/"
+    settings = 'os', 'compiler', 'build_type', 'arch'
```

Comment:
```suggestion
    settings = "os', "compiler", "build_type", "arch"
    provides = "breakpad"
```

Please use double quotes everywhere + add `provides` because `sentry` also maintains a breakpad fork.